windows-vagrant/playbook.yml

```
@@ -0,0 +1,24 @@
+---
+- name: GitHub Actions Runner
+  hosts: all
+  become: yes
+  vars:
+    - runner_user: runner
+  tasks:
+    - name: Runner User
+      user:
+        name: '{{ runner_user }}'
+        state: present
+    - name: Setup GitHub actions runner
+      include_role:
+        name: monolithprojects.github_actions_runner
+      vars:
+        access_token: "{{ lookup('env','PAT') }}"
+      tags:
+        - uninstall
+    - name: Setup Docker engine
+      include_role:
+        name: nickjj.docker
+      vars:
+        docker__users:
+          - '{{ runner_user }}'
```

We noticed that installing Docker after runner can make runner error on docker usage, can you move docker task before of runner. Also we added passwordless sudo and apt update in hpc playbook can you do the same.

```suggestion
    - name: Passwordless sudo for {{ runner_user }} user 
      lineinfile:
        path: /etc/sudoers.d/{{ runner_user }}
        line: '{{ runner_user }} ALL=(ALL) NOPASSWD: ALL'
        state: present
        mode: 0440
        create: yes
        validate: 'visudo -cf %s'
    - name: Only run "update_cache=yes" if the last one is more than 3600 seconds ago
      apt:
        update_cache: yes
        cache_valid_time: 3600        
    - name: Setup Docker engine
      include_role:
        name: nickjj.docker
      vars:
        docker__users:
          - '{{ runner_user }}'
    - name: Setup GitHub actions runner
      include_role:
        name: monolithprojects.github_actions_runner
      vars:
        access_token: "{{ lookup('env','PAT') }}"
        runner_dir: /opt/actions-runner/{{ github_account }}/{{ github_repo }}
      tags:
        - uninstall
```